docs/release-notes/release-notes-0.15.0.md

```diff
@@ -98,6 +98,9 @@
 * [rpc-check fails if it finds any changes](https://github.com/lightningnetwork/lnd/pull/6207/)
   including new and deleted files.
 
+* [The `golangci-lint` package was updated and new linters were
+  enabled](https://github.com/lightningnetwork/lnd/pull/6244).
```

Comment:
nit: we may want to mention for local dev we need to upgrade the `golangci-lint` or delete the old `golangci-lint`.

Reply:
Good point! Added that to the release notes.